docs/ycsb.md

```diff
@@ -50,16 +50,29 @@ Note: We currently support running either workloade or workloadd( and not in the
 Reason: Similar to reason provided by [YCSB documentation](https://github.com/brianfrankcooper/YCSB/wiki/Core-Workloads#running-the-workloads), we'd need to drop the database
         after running either of the workloadd/workloade. This would mean operator to have knowledge of interaction with the said DB, which was outside the scope of this operator.
 
-`options_run` these are the options passed to ycsb binary while running the workload, this is where the target database and options are passed.
-Please read YCSB documentation on the necessary args required for the particular database. And note, the url or API needs to be accessible from the ycsb pod.
-
 The following options are optional:
 
-`loaded` This can be optionally set to true, if you have already loaded the database.
+`load` This can be optionally set to false, if you'd like to just run the workload and have already loaded previously.
+
+`options_load` This needs to be set if `load` is not defined or set to True, these are the options passed to ycsb binary while running the workload, this is where the target database and options are passed.
```

Comment:
Awkward wording...maybe: "...This needs to be set if `load` is set to true (the default)..."